packages/block-editor/src/components/block-edit/edit.js

```diff
@@ -100,10 +100,10 @@ const EditWithGeneratedProps = ( props ) => {
 			),
 		};
 	}, [
-		name,
 		blockType?.usesContext,
 		blockContext,
 		attributes?.metadata?.bindings,
+		bindableAttributes,
```

Comment:
Missng deps for `useMemo` can result in stale values. The `bindableAttributes` is already used as a dependency for other hooks, so it should be safe to use it here as well.